packages/financial-templates-lib/src/types.ts

```diff
@@ -0,0 +1,14 @@
+import type { AbiItem, toBN } from "web3-utils";
```

Comment:
taken from https://github.com/UMAprotocol/protocol/pull/3315 to minimize merge issues.